[2023-10-17T11:18:59.580+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-11T00:00:00+00:00 [queued]>
[2023-10-17T11:18:59.588+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-11T00:00:00+00:00 [queued]>
[2023-10-17T11:18:59.589+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:18:59.601+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-02-11 00:00:00+00:00
[2023-10-17T11:18:59.607+0000] {standard_task_runner.py:57} INFO - Started process 509 to run task
[2023-10-17T11:18:59.609+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-02-11T00:00:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpg2w64b6m']
[2023-10-17T11:18:59.610+0000] {standard_task_runner.py:85} INFO - Job 46: Subtask waiting_for_data
[2023-10-17T11:18:59.653+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-11T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:18:59.725+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-11T00:00:00+00:00'
[2023-10-17T11:18:59.735+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:18:59.736+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:18:59.736+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:18:59.736+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:18:59.746+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210211T000000, start_date=20231017T111859, end_date=20231017T111859
[2023-10-17T11:18:59.781+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:18:59.797+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:26.589+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-11T00:00:00+00:00 [queued]>
[2023-10-18T10:42:26.601+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-11T00:00:00+00:00 [queued]>
[2023-10-18T10:42:26.601+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:26.634+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-02-11 00:00:00+00:00
[2023-10-18T10:42:26.640+0000] {standard_task_runner.py:57} INFO - Started process 1472 to run task
[2023-10-18T10:42:26.643+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-02-11T00:00:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmphh_ys00i']
[2023-10-18T10:42:26.644+0000] {standard_task_runner.py:85} INFO - Job 45: Subtask waiting_for_data
[2023-10-18T10:42:26.689+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-11T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:26.750+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-11T00:00:00+00:00'
[2023-10-18T10:42:26.767+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:26.771+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210211T000000, start_date=20231018T104226, end_date=20231018T104226
[2023-10-18T10:42:26.780+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 45 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1472)
[2023-10-18T10:42:26.816+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:26.845+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:16:53.780+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-11T00:00:00+00:00 [queued]>
[2023-10-18T11:16:53.790+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-11T00:00:00+00:00 [queued]>
[2023-10-18T11:16:53.790+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:16:53.799+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-02-11 00:00:00+00:00
[2023-10-18T11:16:53.808+0000] {standard_task_runner.py:57} INFO - Started process 210 to run task
[2023-10-18T11:16:53.811+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-02-11T00:00:00+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp8sxmwem5']
[2023-10-18T11:16:53.813+0000] {standard_task_runner.py:85} INFO - Job 44: Subtask waiting_for_data
[2023-10-18T11:16:53.861+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-11T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:16:53.949+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-11T00:00:00+00:00'
[2023-10-18T11:16:53.961+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:16:53.962+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:16:53.962+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:16:53.962+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:16:53.973+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210211T000000, start_date=20231018T111653, end_date=20231018T111653
[2023-10-18T11:16:54.025+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:16:54.056+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:11:33.523+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-11T00:00:00+00:00 [queued]>
[2023-10-19T12:11:33.533+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-11T00:00:00+00:00 [queued]>
[2023-10-19T12:11:33.533+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:11:33.550+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-02-11 00:00:00+00:00
[2023-10-19T12:11:33.560+0000] {standard_task_runner.py:57} INFO - Started process 274 to run task
[2023-10-19T12:11:33.563+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-02-11T00:00:00+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpe5a4es8m']
[2023-10-19T12:11:33.565+0000] {standard_task_runner.py:85} INFO - Job 44: Subtask waiting_for_data
[2023-10-19T12:11:33.623+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-11T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:11:33.709+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-11T00:00:00+00:00'
[2023-10-19T12:11:33.721+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:11:33.725+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:11:33.725+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:11:33.726+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:11:33.736+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210211T000000, start_date=20231019T121133, end_date=20231019T121133
[2023-10-19T12:11:33.777+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:11:33.813+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
